pkg/instrumentation/sdk_test.go

```diff
@@ -730,6 +733,7 @@ func TestInjectDotNet(t *testing.T) {
 						Name:      volumeName,
 						MountPath: "/otel-auto-instrumentation",
 					}},
+					Resources: defaultInitContainerResourceRequirements(),
```

Comment:
Should we also add an e2e test for these?

Reply:
sure :)